plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/AccessTrackingFileSystemFactory.java

```diff
@@ -91,25 +96,28 @@ public TrinoInputFile newInputFile(String location, long length)
         @Override
         public TrinoOutputFile newOutputFile(String location)
         {
-            throw new UnsupportedOperationException();
+            return delegate.newOutputFile(location);
```

Comment:
> `Make TrackingFileSystem delegate operation to wrapped file system`

this would be "delegate  **unsupported**  operations". 
however, i am not convinced we want that. 
`throw new UnsupportedOperationException();` was intentional here: tracking is not implemented yet, so just throw

why not implement proper tracking here?

Reply:
because I don't know how to track deletes etc... but ok I will try